### PR TITLE
Help Center: Add title and icons to minimized pages

### PR DESCRIPTION
**Pull request by escapemanuele**
#### Changes proposed in this Pull Request
Context: p1652194080777319/1652191736.493179-slack-C02T4NVL4JJ
image

#### Testing instructions

Added the page title to the minimized help center header: 

* checkout
* `yarn dev --sync` from ETK
* open the help center
* verify that the titles match the screenshot above

### CODE REVIEW

File: client/blocks/inline-help/inline-help-center-content.jsx [hunk 1]
```diff
@@ -44,7 +48,20 @@ const InlineHelpCenterContent = ( {
 		if ( contentTitle ) {
 			contentTitle.focus();
 		}
-	}, [ activeSecondaryView ] );
+
+		if ( activeSecondaryView === VIEW_CONTACT ) {
+			setHeaderText( __( 'Contact our WordPress.com experts', 'full-site-editing' ) );
```
Comment (alshakero) [inline client/blocks/inline-help/inline-help-center-content.jsx]: You don't need the `'full-site-editing'`in Calypso.

**Review by alshakero — COMMENTED** [review]
The code duplication and the drilling is somewhat unpleasant. With your permission, I pushed another solution that we can even extend for articles. But didn't want to make the PR bigger.

**Review by alshakero — APPROVED** [review]
Reviewed and tested.

**Comment by poligilad-auto** [comment]
Nicely done! 

Only thing is:
- make the icons smaller: 20px
- Add a bit of spacing between the icon and the title: 4px
- The chat one should had a "chat" (commentContent) icon.
image

**Pull request merged**